ATL/schema.sql

```diff
@@ -0,0 +1,69 @@
+Create Table Passengers(

```

Comment:
Although this is a preference thing, but the standard convention is to use ALL CAPS when writing sql keywords like CREATE, PRIMARY KEY, WHERE, etc...